Dockerfile

```diff
@@ -49,11 +49,13 @@ RUN apt-get -qq -y update && \
     apt-get install -y --no-install-recommends \
     gnupg1 \
     curl \
+    wget \
     ca-certificates \
     lsb-release && \
     release=`lsb_release -sc` && \
     # Add repo for MongoDB
-    curl "https://www.mongodb.org/static/pgp/server-${MONGODB_VERSION_REPO}.asc" | apt-key add - && \
+    wget "https://www.mongodb.org/static/pgp/server-${MONGODB_VERSION_REPO}.asc" -O mongo-server-${MONGODB_VERSION_REPO}.asc && \
+    apt-key add mongo-server-${MONGODB_VERSION_REPO}.asc && \
```

Comment:
@sengineer0 @remoteeng00 why the original line does not work? Seems like we should use either curl or wget in this Dockerfile, but no need to use both?

Reply:
Oh, It's return `HTTP/1.1 301 Moved Permanently` so we get empty response, I've changed back to `curl` command.